src/map_io/map_building_packet.cc

```diff
@@ -72,11 +72,9 @@ void MapBuildingPacket::read(FileSystem& fs,
 
 						//  Get the tribe and the building index.
 						if (Player* const player = egbase.get_safe_player(p)) {
-							Notifications::publish(NoteMapObjectDescription(
-							   name, NoteMapObjectDescription::LoadType::kObject));
-
```

Comment:
Can this really be removed? It looks like this is only possible, if there is only a single description manager.
Is this guaranteed?
`name` can be replaced by `bd->name()` below, to use the compatibility lookup.

Reply:
There may be more description managers belonging to the Load Game menu's Game instances, but those are not the target for this notification. Only the actual game's description manager actually needs to load the building. So this change slightly increases load times in this case by skipping the unnecessary loading of description duplicates.

Reply:
> So this change slightly increases load times...

Did you mean shorter load times? At least, that's how I understand it.
With your explanation I'm fine with removing it now.

Reply:
Yes I meant decrease :D